ArduCopter/Copter.h

```diff
@@ -912,7 +912,7 @@ class Copter : public AP_Vehicle {
     void Log_Write_Data(LogDataID id, float value);
     void Log_Write_PTUN(uint8_t param, float tuning_val, float tune_min, float tune_max, float norm_in);
     void Log_Video_Stabilisation();
-    void Log_Write_Guided_Position_Target(ModeGuided::SubMode submode, const Vector3f& pos_target, bool terrain_alt, const Vector3f& vel_target, const Vector3f& accel_target);
+    void Log_Write_Guided_Position_Target(ModeGuided::SubMode target_type, const Vector3f& pos_target_m, bool terrain_alt, const Vector3f& vel_target_ms, const Vector3f& accel_target_mss);
```

Comment:
not a blocker but I'm not sure it's better to rename "submode" to "target_type" is it?  The enum is not renamed

Reply:
Yep, I agree. I have undone this change.